app/src/main/res/values/dimens.xml

```diff
@@ -485,4 +485,10 @@
 
   <dimen name="home_fragment_padding_top">36dp</dimen>
   <dimen name="home_fragment_padding_bottom">148dp</dimen>
+
+  <dimen name="promoted_story_card_layout_marginEnd">8dp</dimen>
+  <dimen name="promoted_story_card_layout_marginStart">8dp</dimen>
+  <dimen name="promoted_story_card_layout_width">-1px</dimen>
```

Comment:
This is incorrect for two reasons:
1. We should not give negative values to any dimens value.
2. All values should be in `sp` and `dp` only.

Reply:
@rt4914 sir I had doubt regarding this dimension as  the promoted_story_card_width needed to be set as match parent, I was not getting how to set it and on google I found this as an answer, can you pls help with whats the alternative of this.

Reply:
https://stackoverflow.com/questions/16478808/value-equals-to-match-parent-or-fill-parent-in-dimens-xml   I used this as a reference

Reply:
done